### ISSUE TITLE
Design "Tx validation" section

### ISSUE THREAD

**Issue by alegadea** [txpipe/laceanatomy#7]
Each time we dissect a transaction, we will run each individual validation involved in the complete transaction validation, for instance for Babbage we have [these](https://github.com/txpipe/pallas/blob/00ece5d3008c1e808fb700914992853ece37a06b/pallas-applying/src/babbage.rs#L41-L56). The objective is to provide information on which checks have passed and which have not, for these last ones also give the reason of failure.

So, the following is just something really sketchy to serve as an example of what maybe we want:
![image](https://github.com/txpipe/laceanatomy/assets/73836359/742ac495-7eb6-465d-bac5-d146045aed6a)

Basically, it is an itemized list with checks. **We could start with this but in the near future move to some table with checks or colored boxes.**

**Comment by mikarasv** [comment]
Solved on PR #8
